dcos/util.py

```diff
@@ -92,6 +92,22 @@ def ensure_dir(directory):
                 'Cannot create directory [{}]: {}'.format(directory, e))
 
 
+def ensure_file(path):
```

Comment:
It was consistent with the existing `ensure_dir` function.  I changed both to `*_exists`.